single_stage_detector/ssd/train.py

```diff
@@ -110,7 +110,8 @@ def coco_eval(model, coco, cocoGt, encoder, inv_map, threshold,
             if use_cuda:
                 inp = inp.cuda()
             ploc, plabel = model(inp)
-
+            ssd_print(key=mllog_const.MAX_SAMPLES,
```

Comment:
I believe the key `mllog_const.MAX_SAMPLES` is supposed to be the replacement for the old v0.5 `mlperf_log.NUM_CROPPING_ITERATIONS` (that used to be logged at https://github.com/mlperf/training/pull/367/files#diff-e52cd97e4fbc7eaac24a2c0d3d71c46aL306 )
so I think this `ssd_print` statement should be moved there and the `value` should be `self.num_cropping_iterations` from that function, rather than `len(ploc)`.